src/modules/Account/Profile.ts

```diff
@@ -140,6 +140,21 @@ class Profile extends TagoIOModule<GenericModuleParams> {
     return result;
   }
 
```

Comment:
This can cause backward compatibility problems.  If you are renaming the function, create auditLog function that call auditLogCreate. If the old function is deprecated throw a message